_compress_images_cache.yml

```diff
@@ -1524,122 +1524,6 @@ assets/images/projects/projects@2x.jpg: projects@2x.jpg
-assets/images/guide/designing-products/accessibility/accessibility@2x.jpg: accessibility@2x.jpg
-assets/images/guide/designing-products/accessibility/annotations-bad-big.png: annotations-bad-big.png
-assets/images/guide/designing-products/accessibility/annotations-bad-isolated-big.png: annotations-bad-isolated-big.png
-assets/images/guide/designing-products/accessibility/annotations-good-big.png: annotations-good-big.png
-assets/images/guide/designing-products/accessibility/annotations-good-isolated-big.png: annotations-good-isolated-big.png
-assets/images/guide/designing-products/accessibility/ordering-bad-big.png: ordering-bad-big.png
-assets/images/guide/designing-products/accessibility/ordering-good-big.png: ordering-good-big.png
-assets/images/guide/glossary/address/address-compact.png: address-compact.png
-assets/images/guide/glossary/address/address-compact@2x.png: address-compact@2x.png
-assets/images/guide/glossary/address/address-expanded.png: address-expanded.png
-assets/images/guide/glossary/address/address-expanded@2x.png: address-expanded@2x.png
-assets/images/guide/glossary/address/address-invalid.png: address-invalid.png
-assets/images/guide/glossary/address/address-invalid@2x.png: address-invalid@2x.png
-assets/images/guide/glossary/address/address-valid.png: address-valid.png
-assets/images/guide/glossary/address/address-valid@2x.png: address-valid@2x.png
-assets/images/guide/glossary/address/address-reuse.png: address-reuse.png
-assets/images/guide/glossary/address/address-reuse@2x.png: address-reuse@2x.png
-assets/images/guide/resources/design-files/bitcoin-for-local-business-trifold.png: bitcoin-for-local-business-trifold.png
-assets/images/guide/resources/design-files/bitcoin-for-local-business-trifold@2x.png: bitcoin-for-local-business-trifold@2x.png
-assets/images/guide/daily-spending-wallet/sending/confirm-high-fees.png: confirm-high-fees.png
-assets/images/guide/daily-spending-wallet/sending/confirm-high-fees@2x.png: confirm-high-fees@2x.png
-assets/animations/astronaut/images/img_0.png: img_0.png
-assets/animations/astronaut/images/img_1.png: img_1.png
-assets/animations/contribute/formatting/bitcoin-design-seal-miami-edition.png: bitcoin-design-seal-miami-edition.png
-assets/animations/contribute/formatting/bitcoin-design-seal.png: bitcoin-design-seal.png
-assets/animations/contribute/formatting/bitcoin-design-seal-miami-edition@2x.png: bitcoin-design-seal-miami-edition@2x.png
-assets/animations/contribute/formatting/bitcoin-design-seal@2x.png: bitcoin-design-seal@2x.png
-assets/images/home/banner/bits-and-pieces-fallback.png: bits-and-pieces-fallback.png
-assets/images/home/banner/bits-and-pieces-fallback@2x.png: bits-and-pieces-fallback@2x.png
-assets/images/guide/resources/design-files/bitcoin-wallet-backup-print-template.jpg: bitcoin-wallet-backup-print-template.jpg
-assets/images/guide/resources/design-files/bitcoin-wallet-backup-print-template@2x.jpg: bitcoin-wallet-backup-print-template@2x.jpg
```

Comment:
Same comment I made on #876. This is a lot of removed lines. Seems like an error.

Reply:
Double-checked, and this just removes duplicate lines. The script does it automatically for me. Not sure how these duplicate lines sneak in there, maybe when someone manually resolves conflicts?